### PR TITLE
Add component to autogenerate multi-language code blocks

### PR DESCRIPTION
**Pull request by ismailmustafa**

### CODE REVIEW

File: src/components/MultiLanguageCodeBlock.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+  return <div>
+    {response === null ? <ClipLoader
+      color="#ffffff"
+      loading={true}
+      size="25px"
+      aria-label="Loading Spinner"
+      data-testid="loader"
+    /> :
+      <Tabs groupId="api-token-platform" queryString>
```
Comment (turnipdabeets) [inline src/components/MultiLanguageCodeBlock.js:100]:
```suggestion
      <Tabs groupId="multi-language-platform" queryString>
```

File: src/components/MultiLanguageCodeBlock.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  return <div>
+    {response === null ? <ClipLoader
+      color="#ffffff"
+      loading={true}
+      size="25px"
+      aria-label="Loading Spinner"
+      data-testid="loader"
+    /> :
+      <Tabs groupId="multi-language-platform" queryString>
+        <TabItem value="javascript" label="Curl">
```
Comment (turnipdabeets) [inline src/components/MultiLanguageCodeBlock.js:101]:
```suggestion
        <TabItem value="curl" label="Curl">
```

File: src/components/MultiLanguageCodeBlock.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+        <TabItem value="csharp" label="CSharp">
+          <CodeBlock
+            language="csharp"
+            title={title}
+            text={response?.csharp}
+          ></CodeBlock>
+        </TabItem>
+        <TabItem value="dart" label="Dart">
+          <CodeBlock
+            language="dart"
```
Comment (turnipdabeets) [inline src/components/MultiLanguageCodeBlock.js:117]: does dart work or should this be javascript?
```suggestion
            language="javascript"
```

Comment (Will5) [reply src/components/MultiLanguageCodeBlock.js:117]: It works in my editor, but not in Docusaurus. It works in GitHub, so I wonder if it is a Docusaurus limitation 🤔 

Examples:

```javascript
import { Embedded } from '@beyondidentity/bi-sdk-js';

const embedded = await Embedded.initialize();
```

```kotlin
import com.beyondidentity.embedded.sdk.EmbeddedSdk

EmbeddedSdk.init(
    app: Application,
    keyguardPrompt: (((allow: Boolean, exception: Exception?) -> Unit) -> Unit)?,
    logger: (String) -> Unit,
    biometricAskPrompt: String, /* Optional */
    allowedDomains: List<String>?, /* Optional */
)
```

```swift
import BeyondIdentityEmbedded

Embedded.initialize(
    allowedDomains: [String] = ["beyondidentity.com"],
    biometricAskPrompt: String,
    logger: ((OSLogType, String) -> Void)? = nil,
    callback: @escaping(Result<Void, BISDKError>) -> Void
)
```

```javascript
import { Embedded } from '@beyondidentity/bi-sdk-react-native';

Embedded.initialize(
  biometricAskPrompt: string,
  allowedDomains?: string[]
): Promise<Success>;
```

```dart
import 'package:bi_sdk_flutter/embeddedsdk.dart';

EmbeddedSdk.initialize(
    String biometricAskPrompt,
    List<String>? allowedDomains, /* Optional */
    Future<Function>? logger, /* Optional */
)
```

Comment (Will5) [reply src/components/MultiLanguageCodeBlock.js:117]: Yes and no! We have to add these languages to the prism config. I'm going to make a PR to add dart, kotlin and swift

Comment (turnipdabeets) [reply src/components/MultiLanguageCodeBlock.js:117]: oh nice! I wonder if we need to add csharp, ruby, rust, and java to that list. I haven't tried those yet but node, go, and python seem to be ok.

**Review by turnipdabeets — APPROVED** [review]

**Pull request merged**